crates/ty_python_semantic/resources/mdtest/generics/legacy/paramspec.md

```diff
@@ -244,16 +244,30 @@ Explicit specialization of a generic class involving `ParamSpec` is done by prov
 of types, `...`, or another in-scope `ParamSpec`.
 
 ```py
+reveal_type(OnlyParamSpec[[]]().attr)  # revealed: () -> None
 reveal_type(OnlyParamSpec[[int, str]]().attr)  # revealed: (int, str, /) -> None
+reveal_type(OnlyParamSpec[(int, str)]().attr)  # revealed: (int, str, /) -> None
```

Comment:
you could add a comment here saying that this is "poor taste", and that users shouldn't write their annotations like this. But that we allow it anyway because `OnlyParamSpec[(int, str)]` has the same AST as `OnlyParamSpec[int, str]` (according to CPython, at least), and because mypy/pyright allow it